### ISSUE TITLE
Mangrove Coastal Protection Data

### ISSUE THREAD

**Issue by klongleywood** [Vizzuality/mangrove-atlas#718]
@dhakelila (not sure who else will be working on this) and fyi @lhilarides 

See the attached csv for the Mangrove Coastal Protection Stats.  There are 3 statistics (Area Protected in km2, people protected, and total stock protected, in $USD) for each storm scenario (Annual Expected Benefits, Benefits from a 1-in-25-Year Storm and Benefits from a 1-in-100-Year Storm).  Note that there are values per country and then worldwide values

We may want explore rounding the values, and possibly excluding countries with very small mangrove tracts -- we will discuss on our next data team call, but in the meantime this should get you what you need to get started on this widget. Let me know if there are questions!

[Mangrove_Coastal_Protection_Stats_20230426.csv](https://github.com/Vizzuality/mangrove-atlas/files/11336782/Mangrove_Coastal_Protection_Stats_20230426.csv)

**Comment by dhakelila** [comment]
Thanks, @klongleywood , adding also @AngelArcones to the conversation as he will be the one starting to process the data. 

This is for the "Flood Protection" widget, right?

**Comment by klongleywood** [comment]
@dhakelila yes, Flood Protection -- thanks for clarifying!

**Comment by AngelArcones** [comment]
@klongleywood I've been reviewing the data from the original paper on mangrove flood protection, and I've noticed that in their national aggregation they show benefits per hectare of mangrove, in addition to absolute values per country. We are currently working with the absolute national values, and this works fine for the widgets, but if we want to show them on the map they would just highlight the bigger countries (higher absolute values) over smaller ones.
Perhaps we could use those normalised values by ha for the map, so that countries' values can be compared between them, and it narrows the overall range of the scale. We could also discuss if the widgets should be on absolute or normalised values (maybe have a switcher between options).

However, I have two inquiries on this approach: the first one is whether it's ok from your side to use that data, or if you want us to restrict to absolute values. The second one is that this data at national level is only available for "annual" scenario. However, we could calculate values for the other scenarios easily (using simple math), or maybe you can request that data for all scenarios from your side.

Let us know your thoughts and preferences about this and we'll move forward accordingly

**Comment by klongleywood** [comment]
Hi @AngelArcones  -- thanks for looking into this! Why don't I discuss with the data team on our Friday call to discuss. I'll let you know then what we decide.

**Comment by klongleywood** [comment]
Hi @AngelArcones -- we just discussed on our team call and we agreed to keep the absolute values.  Normalizing the values can introduce some other issues especially comparing countries with very small and large populations. This study also didn't use GMW so the extents we'd also run into some consistency issues there. 

Thanks for checking on this!

**Comment by mluena** [comment]
@AngelArcones can we close this issue?

**Comment by AngelArcones** [comment]
@mluena Yes, this data was aready implemented and has been fine since then, so we can close this
